SIGs/getting-started-guide.md

```diff
@@ -0,0 +1,46 @@
+- [ ] The TSC will consider the proposal and make a decision in a reasonable timeframe. It may adopt the proposal as made, but it can also modify the proposal if it believes it necessary. If approved the TSC will formally appoint one or more Chairs, who should follow the steps below to establish the SIG. 
+
+## Establishing a SIG
+
+Bytecode Alliance SIGs operate transparently and in full public view, open to participation by all. Approved SIGs therefore require a variety of resources to support their operation. For help with any of these contact David Bryant, Bytecode Alliance Executive Director, at david@bytecodealliance.org
+
+- [ ] Create a Google Group for the SIG within the Bytecode Alliance Google Workspace, allowing easy communication and sharing of materials among SIG members
+- [ ] Create a subfolder for the SIG in the SIGs folder of the Bytecode Alliance [governance](https://github.com/bytecodealliance/governance/tree/main/SIGs) repository 
+    - [ ] Add the approved proposal for the SIG to its subfolder, making sure the stated scope, goals, and members are up to date.
+    - [ ] Update the README in the overall SIGs subfolder to include the new SIG with its Chair(s).
+- [ ] Ensure SIG Chairs are added to the [`sig-admin`](https://groups.google.com/u/1/a/bytecodealliance.org/g/sig-admin) Google Group to get admin information and access, such as in organizing Zoom meetings, access meeting recordings, update the Bytecode Alliance public calendar, etc.
+- [ ] Create a  channel for the SIG on the Bytecode Alliance [`Zulip server`](https://bytecodealliance.zulipchat.com)
+- [ ] Add SIG Chairs as maintainers to the Bytecode Alliance [`meetings`](https://github.com/bytecodealliance/meetings) repository so they can manage a subfolder there for the SIG's meeting agenda, notes, presentations, etc.
+    - [ ] Create the subfolder for the SIG, and add a README that summarizes general information about attending SIG meetings.
+    - [ ] Update the README in the [`meetings`](https://github.com/bytecodealliance/meetings) repo to include the new SIG. As soon as the SIG Chair(s) are appointed please identify them in the repo README as well.
+    - [ ] Add meeting agegnda and notes to the subfolder as meetings are planned and occur, typically organized by year.
+- [ ] Add SIG meetings to the Bytecode Alliance [public calendar](https://calendar.google.com/calendar/embed?src=events%40bytecodealliance.org&ctz=America%2FLos_Angeles). Include important details on joining the meeting or a link to where those details may be found.
+- [ ] Announce the new SIG on Zulip in `#general`
+
+## Running SIG Meetings
+
+The following are minimum requirements of running a SIG:
+
+- All meetings should be public
```

Comment:
```suggestion
- All meetings must be public. The zoom link need not be publicly available, to avoid zoom bombing, but should be shared with anyone who requests it.
```